util/file_test.go

```diff
@@ -89,3 +89,19 @@ func TestPathContainsHiddenFileOrFolder(t *testing.T) {
 		assert.Equal(t, testCase.expected, actual, "For path %s", path)
 	}
 }
+
+func TestExpandArgs(t *testing.T) {
+	testCases := []struct {
+		args     []interface{}
+		expected []interface{}
+	}{
+		{[]interface{}{"1"}, []interface{}{"1"}},
+		{[]interface{}{"words"}, []interface{}{"words"}},
+		{[]interface{}{"something.something\\[0\\]"}, []interface{}{"something.something\\[0\\]"}},
+		{[]interface{}{"-target"}, []interface{}{"-target"}},
+		{[]interface{}{"-lock-timeout=20m"}, []interface{}{"-lock-timeout=20m"}},
+	}
```

Comment:
I wanted to add tests because there were none, but I don't have use cases on the top of my head of expandable args, so I just added those that should not expand. If you have examples I'd be glad to add them.

> Also, if the supplied original arg include the current path, it will be changed even if there is no replacement.

Good catch, I'll make adjustments.

Reply:
You may create a test fixture folder (or use an existing one) and add some tests with more than one parameter that use wildcards and expand to some files.

Reply:
I added an env expansion just for good mesure ;)